package.json

```diff
@@ -4,9 +4,12 @@
   "description": "A platform where people upload add books ",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "nyc --reporter=html --reporter=text-summary mocha -r esm test/**/*.test.js",
```

Comment:
Add this line as a flag here `--reporter=text-lcov` this might be why your codeclimate test coverage isn't working.